sql/privilege.md

```diff
@@ -228,9 +161,124 @@ To be more precise, you can check the privilege information in the `Grant` table
+| Select         | `SelectPriv`     | Reads the table data               |
+| Insert         | `InsertPriv`     | Inserts data to a table             |
+| Update         | `UpdatePriv`     | Updates the table data             |
+| Delete         | `DeletePriv`     | Deleted the table data             |
+| Trigger        | `TriggerPriv`    | /                 |
+| Process        | `ProcessPriv`    | Displays the running task       |
+| Execute        | `ExecutePriv`    | Executes the `EXECUTE` statement       |
+| Drop Role      | `DropRolePriv`   | Executes `DROP ROLE`           |
+| Show View      | `ShowViewPriv`   | Executes `SHOW CREATE VIEW`    |
+| References     | `ReferencesPriv` | /                |
+| Create View    | `CreateViewPriv` | Creates a View                 |
+| Create User    | `CreateUserPriv` | Creates a user                |
+| Create Role    | `CreateRolePriv` | Executes `CREATE ROLE`         |
+| Show Databases | `ShowDBPriv`     | Shows the table status in the database |
+
+### ALTER
+
+- For all the `ALTER` statement, users must have the `ALTER` privilege for the corresponding table.
+- For statements except `ALTER...DROP` and `ALTER...RENAME TO`, users must have the `INSERT` and `CREATE` privileges for the corresponding table.
+- For the `ALTER...DROP` statement, users must have the `DROP` privilege for the corresponding table.
+- For the `ALTER...RENAME TO` statement, users must have the `DROP` privilege for the table before renaming, and the `CREATE` and `INSERT` privileges for the table after renaming.
+
+> **Note:**
+>
+> In MySQL 5.7 documentation, users need `INSERT` and `CREATE` privileges to perform the `ALTER` operation on a table. But in reality for MySQL 5.7.25, only the `ALTER` privilege is required in this case. Currently, the `ALTER` privilege in TiDB is consistent with the actual behavior in MySQL.
+
+### CREATE DATABASE
+
+Requires the `CREATE` privilege for the database.
+
+### CREATE INDEX
+
+Requires the `INDEX` privilege for the table.
+
+### CREATE TABLE
 
-#### Grant table
+Requires the `CREATE` privilege for the table.
+
+To execute the `CREATE TABLE...LIKE...` statement, the `SELECT` privilege for the table is required.
+
+### CREATE VIEW
+
+Requires the `CREATE VIEW` privilege.
+
+> **Note:**
+>
+> If the current user is not the user that creates the View, he needs both the `CREATE VIEW` and `SUPER` privileges.
+
+### DROP DATABASE
+
+Requires the `DROP` privilege for the table.
+
+### DROP INDEX
+
+Requires the `INDEX` privilege for the table.
+
+### DROP TABLES
+
+Requires the `DROP` privilege for the table.
+
+### TRUNCATE TABLE
+
+Requires the `DROP` privilege for the table.
+
+### RENAME TABLE
+
+Requires the `ALTER` and `DROP` privileges for the table before renaming and the `CREATE` and `INSERT` privileges for the table after renaming.
+
+### ANALYZE TABLE
+
+Requires the `INSERT` and `SELECT` privileges for the table.
+
+### SHOW
+
+`SHOW CREATE TABLE` requires any of the privileges.
```

Comment:
Does this mean all privileges or any one of the privileges?

Reply:
@lilin90 Any one of the privileges.